src/backend/src/services/change-requests.services.ts

```diff
@@ -149,8 +149,15 @@ export default class ChangeRequestsService {
       }
     });
 
+    const newCR = await prisma.change_Request.findUnique({
```

Comment:
Why do you need to make a new variable instead of just using the updated changed request above, and editing its include args?

Reply:
I make a new variable as changing the updated variable's include args will mess with the sendSlackCRStatusToThread function call which uses updated.notificationSlackThreads.

Reply:
You can have the notificationSlackThreads and also the change request query args in the same include statement